### ISSUE TITLE
`sbot invite.addMe` does not work

### ISSUE THREAD

**Issue by myf** [ssbc/scuttlebot#312]
according to the [docs](https://ssbc.github.io/scuttlebot/plugins/invite.html) `sbot invite.addMe` is an alias to `sbot invite.accept`. when i execute this command with a valid invitation code I got the following error:

> node sbot.js invite.addMe [omitting actual invite code]

```
{ message: 'this.accept is not a function',
  name: 'TypeError',
  stack: 'TypeError: this.accept is not a function
    at EventEmitter.<anonymous> (/home/myf/modules/scuttlebot/plugins/invite.js:120:21)
    at apply (/home/myf/modules/scuttlebot/node_modules/muxrpc-validation/index.js:173:15)
    at EventEmitter.<anonymous> (/home/myf/modules/scuttlebot/node_modules/muxrpc-validation/index.js:82:14)
    at EventEmitter.hooked (/home/myf/modules/scuttlebot/node_modules/hoox/index.js:10:15)
    at Object.PacketStream.request (/home/myf/modules/scuttlebot/node_modules/secret-stack/node_modules/muxrpc/index.js:102:27)
    at PacketStream._onrequest (/home/myf/modules/scuttlebot/node_modules/packet-stream/index.js:149:17)
    at PacketStream.write (/home/myf/modules/scuttlebot/node_modules/packet-stream/index.js:122:41)
    at /home/myf/modules/scuttlebot/node_modules/secret-stack/node_modules/muxrpc/pull-weird.js:55:15
    at /home/myf/modules/scuttlebot/node_modules/secret-stack/node_modules/pull-stream/sinks.js:15:33
    at /home/myf/modules/scuttlebot/node_modules/pull-stream/throughs.js:73:11' }
Alias for accept.
```

**Comment by dominictarr** [comment]
does it work if you use `accept`?

**Comment by myf** [comment]
@dominictarr yes

**Comment by dominictarr** [comment]
okay lets just remove `addMe` then, @pfraze what do yo think?

**Comment by myf** [comment]
ah i think the problem is the scope, one sec

**Comment by myf** [comment]
it's this line here https://github.com/ssbc/scuttlebot/blob/master/plugins/invite.js#L120
where this has a different scope? this.invite.accept might make sense but i don't quite understand what's going on here

**Comment by pfraze** [comment]
@myf's PR fixes it, but it might be a good idea to cut the function.

**Comment by dominictarr** [comment]
obviously no one else was using it because otherwise we would have heard about this issue already.
lets just remove it.

**Comment by myf** [comment]
I am cool with that too, a bit redundant. This is to show `this` was not
used properly.

**Comment by dominictarr** [comment]
cool, will remove then.

**Comment by dominictarr** [comment]
fixed in 7.6.2
